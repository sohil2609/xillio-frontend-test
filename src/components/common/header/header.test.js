import React from 'react';
import { shallow } from 'enzyme';
import Header from './header.component';

const setUp = (props = {}) => {
	const component = shallow(<Header {...props} />);
	return component;
};

describe('Header Component', () => {
	let component;
	let mockProps;
	beforeEach(() => {
		mockProps = {
			title: 'xillio'
		};
		component = setUp(mockProps);
	});

	it('Should render without errors', () => {
		const wrapper = component.find('[data-test="headerComponent"]');
		expect(wrapper.length).toBe(1);
	});

	it('Should test snapshot', () => {
		expect(component).toMatchSnapshot();
	});
});
