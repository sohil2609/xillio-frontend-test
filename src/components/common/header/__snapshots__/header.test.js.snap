// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Should test snapshot 1`] = `ShallowWrapper {}`;
